frontend/src/Editor/WidgetManager/widgetConfig.js

```diff
@@ -499,6 +507,7 @@ export const widgets = [
         loaderColor: { value: '' },
         visibility: { value: '{{true}}' },
         borderRadius: { value: '{{0}}' },
+        borderColor: { value: '' },
```

Comment:
Can we add a default value for the border color?
```suggestion
        borderColor: { value: '#375FCF' },
```

Reply:
other than that LGTM!
@vedant-z 🚀